packages/frontend/src/components/common/ResourceView/index.tsx

```diff
@@ -84,7 +84,7 @@ const ResourceView: React.FC<ResourceViewProps> = ({
         type: ResourceViewItemAction.CLOSE,
     });
 
-    const [activeTabId, setActiveTabId] = useState(tabs?.[0]?.id);
+    const [activeTabId, setActiveTabId] = useState(tabs?.[3]?.id);
```

Comment:
im very confused. how does this solve the problem? I mean, why it is a problem on the first place

Reply:
what happens if you refresh (hard refresh) the page. does it then show up the chart item? if so, then this is a caching issue

Reply:
ok, we should probably fix the root cause because if we add/remove tabs this can slip out yet as another bug.

Reply:
you mean the logic of `slicedSortedItems`

Reply:
done 💯